src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -118,6 +118,7 @@ const ReleaseServiceStatus = ({
       )}
 
       {currentStatus &&
+        currentStatus.overallStage !== 'Validating' &&
```

Comment:
We know there are specific `overallStage` values, so we should at least define these in the `ReleaseStatus` rather than use a string.

Your earlier condition alluded to `ReleaseStatus` having more than just an `overallStatus` as well. Why aren't these included in the type?